applications/OptimizationApplication/custom_utilities/response/linear_strain_energy_response_utils.cpp

```diff
@@ -311,7 +318,7 @@ void LinearStrainEnergyResponseUtils::CalculateStrainEnergyLinearlyDependentProp
     const auto& r_process_info = rModelPart.GetProcessInfo();
 
     block_for_each(rModelPart.Elements(), tls_type(), [&](auto& rElement, tls_type& rTLS) {
-        if (rElement.Is(SELECTED) && rElement.IsActive()) {
+        if (rElement.IsActive()) {
```

Comment:
do we still need isActive?! I would say with the new handling of the sensitivity model parts we do not need.

Reply:
I would keep it, so to be consistent with the builder and sovlers.